WPMedia/arc-themes-components
# Lint Updates

## Ticket

- [THEMES-1460](https://arcpublishing.atlassian.net/browse/THEMES-1460)

## Description

Add back in linting

## Acceptance Criteria

eslint is running without file errors (may need to determine cause)
Git action (husky hook) - linters are running on pre-commit (uncomment line)

## Test Steps

1. Checkout branch - `git checkout themes-1460`
2. Update dependencies - `npm i`
3. Run Storybook `npm run lint`
4. Ensure linting passes (with warnings)
5. Run Storybook `npm run lint:styles`
6. Ensure linting passes
7. Update a file and stage it
8. Should see no errors (unless you created one)

## Author Checklist


## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1460]: https://arcpublishing.atlassian.net/browse/THEMES-1460?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
